src/Parsing/Impl/Ast/FStringExpression.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Python.Parsing.Ast {
+    public class FStringExpression : Expression {
+        public List<Expression> Children { get; }
```

Comment:
Can we reuse `IEnumerable<Node> GetChildNodes()` ? Having two similar methods is confusing.